src/vision/status_quo/barbara_tries_unix_socket.md

```diff
@@ -0,0 +1,314 @@
+# 😱 Status quo stories: Template
+
+*This is a template for adding new "status quo" stories. To propose a new status quo PR, do the following:*
+
+* *Create a new file in the [`status_quo`] directory named something like `Alan_tries_to_foo.md` or `Grace_does_bar.md`, and start from [the raw source from this template]. You can replace all the italicized stuff. :)*
+* *Do not add a link to your story to the [`SUMMARY.md`] file; we'll do it after merging, otherwise there will be too many conflicts.*
+
+*For more detailed instructions, see the [How To Vision: Status Quo] page!*
+
+*If you're looking for ideas of what to write about, take a look at the [open issues]. You can also [open an issue of your own] to throw out an idea for others.*
+
+[How To Vision: Status Quo]: ../how_to_vision/status_quo.md
+[the raw source from this template]: https://raw.githubusercontent.com/rust-lang/wg-async-foundations/master/src/vision/status_quo/template.md
+[`status_quo`]: https://github.com/rust-lang/wg-async-foundations/tree/master/src/vision/status_quo
+[`SUMMARY.md`]: https://github.com/rust-lang/wg-async-foundations/blob/master/src/SUMMARY.md
+[open issues]: https://github.com/rust-lang/wg-async-foundations/issues?q=is%3Aopen+is%3Aissue+label%3Astatus-quo-story-ideas
+[open an issue of your own]: https://github.com/rust-lang/wg-async-foundations/issues/new?assignees=&labels=good+first+issue%2C+help+wanted%2C+status-quo-story-ideas&template=-status-quo--story-issue.md&title=
+
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+Content of `Cargo.toml` for reproducibility:
+
+```toml
+futures = "0.3.14"
+hyper = { version = "0.14.7", features = ["full"] }
+pretty_env_logger = "0.4.0"
+reqwest = "0.11.3"
+tokio = { version = "1.5.0", features = ["macros", "rt-multi-thread"] }
+```
+
+There is a HTTP server in hyper which Barbara have to query.
+
+```rust
```

Comment:
This block of code is just showing an example HTTP server, right?   And isn't what Barbara has written?  If so, maybe we can omit this (or at least hide it behind a `<details>` tag)

Reply:
Okay, good idea. Maybe we can also hide the `Cargo.toml` within `<details>`.